api/v1alpha1/jobset_types.go

```diff
@@ -130,6 +130,11 @@ type Network struct {
 	// <jobSet.name>-<spec.replicatedJob.name>-<job-index>-<pod-index>.<jobSet.name>-<spec.replicatedJob.name>
 	// +optional
 	EnableDNSHostnames *bool `json:"enableDNSHostnames,omitempty"`
+
+	// Subdomain is an explicit choice for a network subdomain name
+	// When set, the replicated job is added to this network. This would allow sharing common networks
```

Comment:
Here can we include the default subdomain name format that is generated automatically if `Subdomain` is left unset (i.e., `<jobSet.name>-<spec.replicatedJob.name>`)?